### PR TITLE
Update working-with-forms.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Internal review url](https://review.docs.microsoft.com/en-us/aspnet/core/mvc/views/working-with-forms?view=aspnetcore-2.2&branch=pr-en-us-11197#the-form-tag-helper)

### CODE REVIEW

File: aspnetcore/mvc/views/working-with-forms.md [hunk 1]
```diff
@@ -584,16 +584,16 @@ Supported [AnchorTagHelper](xref:mvc/views/tag-helpers/builtin-th/anchor-tag-hel
...
 ### Submit to controller example
 
-The following example submits the form to the `Index` action of `HomeController` if either the input or button were clicked:
+The following code submits the form to the `Index` action of `HomeController` when the input or button are selected:
 
 ```cshtml
 <form method="post">
     <button asp-controller="Home" asp-action="Index">Click Me</button>
-    <input type="image" src="..." alt="Or Click Me" asp-controller="Home" asp-action="Index" />
+    <input type="image" src="..." alt="Alt" asp-controller="Home" asp-action="Index" />
```
Comment (NTaylorMullen) [inline aspnetcore/mvc/views/working-with-forms.md]: Leave as `Or Click Me`. The `<input type="image">` typically relies on its src to say the click me text but obv in this format we don't have that luxury.

Comment (Rick-Anderson) [reply aspnetcore/mvc/views/working-with-forms.md]: I took that out to make to remove the horizontal scroll bar. I'll put that back and make it two lines.

**Comment by Rick-Anderson** [comment]
@NTaylorMullen 	I moved Form Action Tag Helper right after Form Tag Helper and added in content from the Form TH.

Removed future tense.

**Comment by Rick-Anderson** [comment]
@NTaylorMullen 	let me know what I need to change to merge into your PR.

**Comment by NTaylorMullen** [comment]
Feel free to merge whenever. 😄

**Comment by NTaylorMullen** [comment]
Ahh, actually just saw I got some more feedback on the other PR. Let me address that first.

**Comment by NTaylorMullen** [comment]
Done, go for it when you're ready.

**Pull request merged**